07-http_gordle/final/internal/gordle/game.go

```diff
@@ -0,0 +1,107 @@
+package gordle
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// Game holds the information we need to get the Feedback of a play.
+type Game struct {
+	solution []rune
+}
+
+// New returns a Game variable, which can be used to Play!
+func New(corpus []string) (*Game, error) {
```

Comment:
As argument, take the word, it's clearer & easier to test that way. Calling the func with `gordle.New(pickRandomWord(corpus))` should be nicer... similarly, it would allow not require a specific format for the corpus.